tensorflow/examples/udacity/.gitignore

```diff
@@ -0,0 +1,5 @@
+notMNIST.pickle
```

Comment:
Never mind, they're covered by the *. Looks like you still have a stray edit in the .ipynb.